tests/acceptance/bootstrap/SpacesTUSContext.php

```diff
@@ -185,7 +185,11 @@ private function uploadFileViaTus(
 		string $resource,
 		string $spaceName
 	): ResponseInterface {
-		$spaceId = $this->spacesContext->getSpaceIdByName($user, $spaceName);
+		if ($spaceName === 'Shares') {
+			$spaceId = GraphHelper::SHARES_SPACE_ID;
+		} else {
+			$spaceId = $this->spacesContext->getSpaceIdByName($user, $spaceName);
+		}
```

Comment:
`$this->spacesContext->getSpaceIdByName($user, $spaceName);` should be able to get the shares space as well.